api_tests/nodes/views/test_node_institutions_detail.py

```diff
@@ -0,0 +1,29 @@
+from nose.tools import *  # flake8: noqa
+
+from tests.base import ApiTestCase
+from tests.factories import InstitutionFactory, NodeFactory
+
+from api.base.settings.defaults import API_BASE
+
+class TestNodeInstitutionDetail(ApiTestCase):
+    def setUp(self):
+        super(TestNodeInstitutionDetail, self).setUp()
+        self.institution = InstitutionFactory()
+        self.node = NodeFactory(is_public=True)
+        self.node.primary_institution = self.institution
+        self.node.save()
+        self.node2 = NodeFactory()
+
+    def test_return_institution(self):
+        url = '/{0}nodes/{1}/institution/'.format(API_BASE, self.node._id)
+        res = self.app.get(url)
+
+        assert_equal(res.status_code, 200)
+        assert_equal(res.json['data']['attributes']['name'], self.institution.name)
+        assert_equal(res.json['data']['id'], self.institution._id)
+
+    def test_return_no_institution(self):
+        url = '/{0}nodes/{1}/institution/'.format(API_BASE, self.node2._id)
+        res = self.app.get(url, expect_errors=True)
+
+        assert_equal(res.status_code, 401)
```

Comment:
Yep, 401 is for not authorized, which this is returning because no auth is being passed to a private node. Will fix test to pass auth to a node with no institution.